getting-started-with-recoil-in-react-1fca/notes/05_react-use-async-functions-in-a-recoil-selector.md.md

```diff
@@ -0,0 +1,23 @@
+<TimeStamp start="0:07" end="0:12">
+  
+  Selectors can easily be made asynchronous
+  
+</TimeStamp>
+
+<TimeStamp start="2:09" end="2:16">
+  
+  You can use `useRecoilValue` the exact same way you would use it with synchronous selectors
+  
+</TimeStamp>
+
+<TimeStamp start="2:30" end="2:40">
+  
+  [React Suspense](https://reactjs.org/docs/concurrent-mode-suspense.html)
```

Comment:
A short definition of Suspence would be a good idea. The instructor doesn't go into much detail about it at all. I don't think he even mentions that it's currently experimental and subject to change.

Reply:
```suggestion
  "Suspense for Data Fetching is a new feature that lets you also use <Suspense> to declaratively “wait” for anything else, including data." - [reactjs.org](https://reactjs.org/docs/concurrent-mode-suspense.html)
  
 React Suspense is experimental and subject to change.
```